### PR TITLE
NAS-110406 / 21.10 / Improved yarn.lock flow

### PR DESCRIPTION
**Pull request by RehanY147**
NAS-110406

### CODE REVIEW

**Review by undsoft — APPROVED** [review]
So far looks good, but first item in the ticket also needs to be addressed. See `.github/workflows/main.yml`

**Comment by RehanY147** [comment]
> 
> 
> So far looks good, but first item in the ticket also needs to be addressed. See `.github/workflows/main.yml`

Yes, still working on that. The image wasn't building on the CI for some reason. Figuring that out.

**Review by undsoft — APPROVED** [review]
Looks good!

File: package.json [hunk 1]
```diff
@@ -42,88 +42,88 @@
...
+    "q": "~1.5.1",
+    "rxjs": "~6.6.0",
+    "stylefire": "~7.0.2",
+    "what-is-my-ip-address": "~1.0.3",
+    "xterm": "~4.9.0",
+    "xterm-addon-fit": "~0.4.0",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.1000.1",
```
Comment (undsoft) [inline package.json:129]: We can add ~ to devDependencies too.

Comment (RehanY147) [reply package.json:129]: That causes the CI build image to fail for some reason

**Pull request merged**